### PR TITLE
Proof-of-Concept. Notifications groups

### PR DESCRIPTION
**Pull request by sponkrashin**
For issue #232

### CODE REVIEW

File: server/Arcadia.Assistant.Notifications/NotificationsActor.cs [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+    using System.Collections.Generic;
+    using System.Linq;
+
+    using Akka.Actor;
+    using Akka.Event;
+    using Akka.Routing;
+
+    public class NotificationsActor : UntypedActor
+    {
+        private const int NotificationsActorsPoolSize = 5; // Later can be defined in config
```
Comment (santee) [inline server/Arcadia.Assistant.Notifications/NotificationsActor.cs]: let's have it as 1 for now :D

Comment (santee) [inline server/Arcadia.Assistant.Notifications/NotificationsActor.cs]: let's have 1 for now? :)

Comment (santee) [reply server/Arcadia.Assistant.Notifications/NotificationsActor.cs]: Or nevermind :)

File: server/Arcadia.Assistant.Server/Application.cs [hunk 2]
```diff
@@ -67,8 +64,27 @@ public virtual void Dispose()
...
+
+        protected virtual void OnStart(ActorSystem actorSystem)
+        {
+        }
+
+        protected virtual void OnStop(ActorSystem actorSystem)
+        {
+        }
+
+        private void CreateRootActors()
```
Comment (santee) [inline server/Arcadia.Assistant.Server/Application.cs]: please revert it. what's the point to have everything in one place

Comment (sponkrashin) [reply server/Arcadia.Assistant.Server/Application.cs]: It also was in one place before - in ActorSystemBuilder. It also created unused collection that contained all top-level actors.

File: server/Arcadia.Assistant.Notifications/NotificationsActor.cs (hunk 1 above)
Comment (sponkrashin) [reply server/Arcadia.Assistant.Notifications/NotificationsActor.cs]: Up to you, master ;)

**Pull request merged**